flake.nix

```diff
@@ -21,8 +21,8 @@
     devenv.url = "github:cachix/devenv";
     flake-parts.url = "github:hercules-ci/flake-parts";
     homestakeros-base.inputs.nixpkgs.follows = "nixpkgs";
```

Comment:
Is this necessary?

Reply:
As it is a flake, it has nixpkgs pinned to some version X. If it's not updated for a while, it will use the nixpkgs that the flake has pinned, even though here we want to maintain it in sync with ours.

Reply:
Got it